bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml

```diff
@@ -148,7 +148,7 @@ limitations under the License.
                                 <phase>integration-test</phase>
                                 <configuration>
                                     <includes>
-                                        <include>**/IntegrationTests.java</include>
+                                        <include>**/TestKnownHBaseGap.java</include>
```

Comment:
why are we changing this?

Reply:
Previously we only had tests that ran against:
1. both hbase & bigtable
2. just hbase

We added a test that should only run against bigtable.